webpack.base.js

```diff
@@ -17,28 +17,26 @@ module.exports = {
            List all of theme in the array
         */
         test: /\.css$/,
-        include: [/redux-notifications/],
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
-          use: 'css-loader',
-        }),
-      },
-      {
-        /* React-toolbox still relies on SCSS and css-modules */
-        test: /\.scss$/,
-        include: [/react-toolbox/],
+        include: [/redux-notifications/, /react-toolbox/, /normalize.css/],
```

Comment:
The `redux-notifications` styles have to be scoped globally, and this causes them to be prefixed, basically throwing away all default notif styles. I can come up with a fix for this if you want, @erquhart.